src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateInstanceEventHandler.java

```diff
@@ -111,6 +111,7 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload d
                 MappingParameters mappingParameters = Json.decodeValue(mappingMetadata.getMappingParams(), MappingParameters.class);
                 AdditionalFieldsUtil.updateLatestTransactionDate(targetRecord, mappingParameters);
                 AdditionalFieldsUtil.move001To035(targetRecord);
+                AdditionalFieldsUtil.normalize035(targetRecord);
```

Comment:
this function should also be called when Instance is updated

Reply:
good catch, thanks